tests/parser/controllers/joints_motor_minmax_position/joints_motor_minmax_position.c

```diff
@@ -0,0 +1,128 @@
+#include <webots/motor.h>
+#include <webots/robot.h>
+#include <webots/supervisor.h>
+
+#include "../../../lib/ts_assertion.h"
+#include "../../../lib/ts_utils.h"
+
+#define TIME_STEP 32
+
+int main(int argc, char **argv) {
+  ts_setup(argv[0]);
+
+  // This test suite verifies that if the position field, as set in the jointParameters, is outside of the bounds of the
+  // minPosition and maxPosition of RotationalMotor, these bounds must adadpt accordingly.
+  // This suite also checks that the correct pairing is done (device <-> jointParameters, device2 <-> jointParameters2)
+
+  // SliderJoint Tests -------------------------------------------------------------------------------------------------
+  WbDeviceTag sj_motor_lower_lim = wb_robot_get_device("sj_lower_lim_device");
```

Comment:
[No abbreviations](https://github.com/cyberbotics/webots/wiki/Coding-Style#cs44-no-abbreviations): I would replace "sj" with "slider_joint" or "slider" and "lim" with "limit".
I would also replace "device" with "motor" to be more precise.
This comment also applies to other names below.